### ISSUE TITLE
ID 338065: [Programmatic Access - Simon - Table]: Visually appearing table in 'Specific Routing' screen is not programmatically defined as Table

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66358]
### **User Experience:**  
Visually impaired people who depend on screen reader users will not get information about the control If Visually highlighted content which is like table is not programmatically defined under Table due to which they will not be able to understand the purpose of it and cannot perform any corresponding action.
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:**

1. Open #Url: [play.grafana.org](https://play.grafana.org) . Sign in with valid credentials. Home page appears. 
2. Navigate to the left navigation bar and to the "Alerting" button, select it. 
3. It gets expanded and navigate the expanded sub menuitems. 
4. Select the "Alerting" button,new page opens. 
5. Navigate to the "Notifications Policies" tab item and select it. 
6. Navigate the "Notifications Policies" tab. 
7. Navigate the  "Specific Routing" section. Verify the issue.

**Actual Result:** 
Visually appearing table in 'Specific Routing' screen is not programmatically defined as Table so that Screen Reader does not provide the information of row and column and also not associating the header with the row cells.

**Expected Result:** 
Visually appearing table in 'Specific Routing' screen should programmatically defined as Table so that Screen Reader will read the information by associating the column headers with the columns cells
_Note: Same issue appears for throughout the application_

### **Severity:**
`SEV2`

### **Area:**
`DinamicTable Component`
